### ISSUE TITLE
Octopus & Octopi  Word

### ISSUE THREAD

**Issue by ghost** [DiscordDungeons/Bugs#136]
`bug

http://prntscr.com/ego085    - Fising Octopus      
http://prntscr.com/ego17f      - My inv say Octopi    
http://prntscr.com/ego1um    - Global Market

Anyways, Octopus and Octopi mean similar but different. Of course, although interpreted as the same word.
I'd like to use the same word as long as I can. I told you that I caught Octopus through fishing, but I was confused with Octopi. The wiki is also called Octopus.

**Comment by HustlePuff** [comment]
Hello, Octopi is the plural of Octopus, that's why if you have many Octopus in your inventory or on the Global Market it's called Octopi.
Not really a bug :)

**Comment by Wanterwolf** [comment]
Aha? I do not like seafood. But in our country it is written in different words so I wrote it down. I think it is more a mistake than a bug.

Octopus (Our Langs Image) - http://prntscr.com/egr7cr
Octopi - http://prntscr.com/egr6pz

Anyway, I just wrote down the idea that the image on the country where I lived or the country I lived in was a non-English-speaking user, and that I should have written the same words if there was a slight difference. I learn the meaning of the word thanks to it.  :D
